modules/notes/todo-comments/todo-comments.nix

```diff
@@ -1,20 +1,50 @@
-{lib, ...}: let
-  inherit (lib) mkEnableOption mkOption types mkMappingOption;
+{
+  pkgs,
```

Comment:
@NotAShelf any idea why I can't import pkgs here? the error I got:

```
 error: function 'anonymous lambda' called without required argument 'pkgs'

         at /nix/store/ygamy3jl6a9b1cppx5ci73w45ls3ihqv-source/modules/notes/todo-comments/todo-comments.nix:1:1:

              1| {
               | ^
              2|   pkgs,
```

Reply:
in `todo-comments/config.nix:

```diff
- self = import ./todo-comments.nix {inherit lib;};
+ self = import ./todo-comments.nix {inherit pkgs lib;};
```

Reply:
thanks!